buckets.js

```diff
@@ -27,7 +27,8 @@ define('buckets', [], function() {
             // Camel-case it.
             property = property[0].toUpperCase() + property.substr(1);
 
-            for (var i = 0, e; e = prefixes[i++];) {
+            for (var i = 0, e; i < prefixes.length; i++) {
```

Comment:
Good idea to check the theory. 

I tried bumping the array size up to 1000000 in the perf test. That seemed to show for FF that the assignment variant was faster by quite a big margin. However in Chrome the direct comparison seemed faster.

Unless we know we're running huge loops this probably isn't worth worrying about, premature optimisations and all that!